en/flight_modes_rover/index.md

```diff
@@ -1,5 +1,10 @@
 # Flight Modes (Rover)
 
+::: warning
+This page contains flight mode information for differential rovers and the legacy ackermann rover implementation.

```

Comment:
I've called it legacy here, but I need to understand how long it will live. If it is likely gone in v1.16, then we could make this page `Drive Modes (Differential Rovers)`